hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -0,0 +1,146 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.test.web.reactive.server.WebTestClient
+import uk.gov.justice.digital.hmpps.prisonersearch.common.model.SyncIndex
+import uk.gov.justice.digital.hmpps.prisonersearch.search.AbstractSearchDataIntegrationTest
+import uk.gov.justice.digital.hmpps.prisonersearch.search.model.PrisonerBuilder
+import uk.gov.justice.digital.hmpps.prisonersearch.search.repository.PrisonerRepository
+
+class AttributeSearchIntegrationTest : AbstractSearchDataIntegrationTest() {
+
+  private val prisonerBuilders = mutableListOf<PrisonerBuilder>()
+
+  // Although we want the super class to create a new index before the tests run, we don't want to load any data into it, so we can manage prisoners on a test by test basis
+  override fun loadPrisonerData() {}
+
+  private fun PrisonerRepository.delete(prisonerNumber: String) = delete(prisonerNumber, SyncIndex.GREEN)
+
+  private fun WebTestClient.attributeSearch(body: String) =
+    post()
+      .uri("/attribute-search")
+      .headers(setAuthorisation(roles = listOf("ROLE_PRISONER_SEARCH")))
```

Comment:
wondering if we should have a different role for the new IMS search - not sure we want everyone else running it?

Reply:
Yes let's have a chat about this